templates/base.html

```diff
@@ -50,6 +46,9 @@
 {% include "security/_messages.html" %}
 
 <main>
+  <a href={{ url_for('home') }} class=logo>
+    <img src=https://rawgit.com/cafebabel/styleguide/master/img/logo-cafebabel.svg alt="logo Cafébabel">
```

Comment:
You probably want to import this into the project

Reply:
ah oui !